nci/lib/watcher.rb

```diff
@@ -196,6 +196,16 @@ def bump_version
       # the command, meaning we can't just use changelog.new_version :|
       cmd.run(*dch)
 
+    def bump_version_future_is_early
+      changelog = Changelog.new(Dir.pwd)
+      version = Debian::Version.new(changelog.version)
+      version.upstream = newest_version
+      version.revision = '0neon' unless version.revision.to_s.empty?
+      @dch = Debian::Changelog.new_version_cmd(version.to_s, distribution: NCI.future_series, message: 'New release')
+      # A bit awkward we want to give a dch suggestion in case this isn't kde software so we'll want to recycle
+      # the command, meaning we can't just use changelog.new_version :|
+      cmd.run(*dch)
```

Comment:
This function is missing an `end`?

Also, I feel like it may be better to refactor this so we don't have to duplicate the entire function just to switch out the distribution value above. We could just pass in the distribution value as parameter into the function maybe?